meson.build

```diff
@@ -330,8 +335,9 @@ if not headers_only
 	endif
 
 	ld_cpp_args = [
+        '-g',
```

Comment:
These are not to be controlled like this.  See <https://mesonbuild.com/Builtin-options.html#core-options>.